<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rotate</class>
 <widget class="QDialog" name="rotate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>375</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>641</width>
    <height>375</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>641</width>
    <height>375</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rotate</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="10">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Rotation around Point:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Angle:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDial" name="dial">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="spinBox">
     <property name="maximum">
      <number>360</number>
     </property>
    </widget>
   </item>
   <item row="4" column="2" rowspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Rotate Point</string>
     </property>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>rotate</receiver>
   <slot>rotateOKSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>576</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>542</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>548</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>566</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>rotateOKSlot()</slot>
 </slots>
</ui>
